### ISSUE TITLE
"RimCities - Citadel Update" Hostile Cities with no defenders

### ISSUE THREAD

**Issue by wvartech** [cseelhoff/RimThreaded#383]
**Describe the bug**

**Issue 1:
Health Operations do not work,

Example: Operations to cure wounds/scars, remove/install body parts. The patient is sedated(noted on health tab information) but the doctor does not operate. Tested with regular humans and other races, issue reproduceable attempting to remove scars, remove bodyparts.

Regular tending works normally.

**Issue 2:

No NPC's/hostiles on enemy cities.

When entering/attacking a hostile city(negative influence, tested on -100 and -80), no units spawn, and the game declares it as the base being destroyed.

Regular faction bases act as normal, can attack them and they do have hostile units as usual.

**Error Log**
https://gist.github.com/HugsLibRecordKeeper/4280ced28384f79462a46bbcb62e04b0

**Mod List**
Version: 1
Name: Latest2021
Mods:
- Id: brrainz.harmony
  Name: Harmony
- Id: ludeon.rimworld
  Name: Core
- Id: fluffy.modmanager
  Name: Mod Manager
- Id: avius.locks
  Name: Locks
- Id: unlimitedhugs.hugslib
  Name: HugsLib
- Id: orion.hospitality
  Name: Hospitality
- Id: roolo.searchanddestroy
  Name: Search and Destroy
- Id: agentblac.makepawnsprisoners
  Name: 1.2 - Imprisonment On The Go! (Make Pawns Prisoners Without Beds)
- Id: n7huntsman.combatextended.distilling
  Name: Chemical Distilling for Combat Extended
- Id: hatti.qualitybuilder
  Name: QualityBuilder
- Id: erdelf.humanoidalienraces
  Name: Humanoid Alien Races 2.0
- Id: nemonian.my
  Name: Moyo-From the depth
- Id: bep.littlefairy.reboot
  Name: LittleFairy_reboot
- Id: jkviolet.rakkleracemode.copy
  Name: Rakkle the rattle snake Race mod
- Id: jacketgun.ce.rakklepatch
  Name: CE patch for Rakkle the rattle snake Race mod
- Id: gguake.apparelbodyresolver
  Name: Apparel BodyType Resolver
- Id: sarg.magicalmenagerie
  Name: Magical Menagerie
- Id: murmur.enemyselfpreservation
  Name: Enemy Self Preservation
- Id: oblitus.animalslogic
  Name: Animals Logic
- Id: kentington.saveourship2
  Name: Save Our Ship 2
- Id: uuugggg.smartmedicine
  Name: Smart Medicine
- Id: weilbyte.snapout
  Name: Snap Out!
- Id: smashphil.neceros.srtsexpanded
  Name: SRTS Expanded
- Id: jecrell.jecstools
  Name: JecsTools (Unofficial)
- Id: torann.arimworldofmagic
  Name: A RimWorld of Magic
- Id: ceteam.combatextended
  Name: Combat Extended
- Id: n7huntsman.combatextendedweapons
  Name: 'Combat Extended: Weapons'
- Id: ceteam.combatextendedmelee
  Name: Combat Extended Melee
- Id: mlie.callofcthulhucosmichorrors
  Name: Call of Cthulhu - Cosmic Horrors (Continued)
- Id: edb.preparecarefully
  Name: EdB Prepare Carefully
- Id: roolo.giddyupcore
  Name: Giddy-up! Core
- Id: avius.prisonlabor
  Name: Prison Labor
- Id: kato.rimofmadness.vampires
  Name: Rim of Madness - Vampires
- Id: rim.job.world
  Name: RimJobWorld
- Id: babies.and.children
  Name: Babies and Children
- Id: mlie.rimofmadnesswerewolves
  Name: Rim of Madness - Werewolves (Continued)
- Id: linkolas.stabilize
  Name: Stabilize
- Id: dismarzero.vgp.vgpgardentools
  Name: VGP Garden Tools
- Id: petetimessix.simplesidearms
  Name: Simple sidearms
- Id: hc.giantrace
  Name: The GiantRace
- Id: spoonshortage.adogsaidanimalprosthetics
  Name: A Dog Said... Animal Prosthetics
- Id: syrchalis.setupcamp
  Name: '[SYR] Set Up Camp'
- Id: mlie.rimofmadnessarachnophobia
  Name: Rim of Madness - Arachnophobia (Continued)
- Id: ykara.epoe
  Name: Expanded Prosthetics and Organ Engineering
- Id: vanillaexpanded.vfecore
  Name: Vanilla Furniture Expanded
- Id: cabbage.rimcities
  Name: RimCities - Citadel Update
- Id: roolo.giddyupcaravan
  Name: Giddy-up! Caravan
- Id: rah.rbse
  Name: Rah's Bionics and Surgery Expansion
- Id: roolo.giddyupbattlemounts
  Name: Giddy-up! Battle Mounts
- Id: farmradish.nyaron
  Name: Nyaron race
- Id: k.psitech
  Name: PsiTech
- Id: farmradish.nyaronce
  Name: Nyaron race CE Patch
- Id: etrt.rimworldtweaks
  Name: Evil Tactician's RimWorld Tweaks
- Id: moretraitslots.kv.rw
  Name: '[KV] More Trait Slots'
- Id: lwm.deepstorage
  Name: LWM's Deep Storage
- Id: jangodsoul.simplestorage
  Name: '[JDS] Simple Storage'
- Id: gloomy.gloomyhairmk2
  Name: Gloomy Hair mk2
- Id: velc.hatsdisplayselection
  Name: Hats Display Selection
- Id: fluffy.colonymanager
  Name: Colony Manager
- Id: mlie.callofcthulhucults
  Name: Call of Cthulhu - Cults (Continued)
- Id: brrainz.cameraplus
  Name: Camera+
- Id: solaris.ratkinracemod
  Name: NewRatkinPlus
- Id: shavius.alienrbse
  Name: Ratkin/Rabbie/Dragonian/Moyo RBSE compatibility patch
- Id: crlsniper.realisticrooms
  Name: Realistic Rooms
- Id: sarg.geneticrim
  Name: Genetic Rim
- Id: sarg.geneticrimcosmichorrors
  Name: GeneticRim Cosmic Horrors Patch
- Id: caesarv6.damageindicators
  Name: Damage Indicators [1.2]
- Id: proxyer.desurgeries.desurgeries
  Name: '[1.2] DE Surgeries'
- Id: proxyer.desurgeries.newratkinpatch
  Name: '[1.2] DE Surgeries - NewRatkinPlus patch'
- Id: roo.antyracemod
  Name: Anty the war ant race
- Id: dhultgren.buryonthespot
  Name: Bury On The Spot
- Id: falconne.aff
  Name: Avoid Friendly Fire
- Id: avius.badpeople
  Name: Bad People
- Id: neceros.badcanbegood
  Name: Bad Can Be Good
- Id: majorhoff.rimthreaded
  Name: RimThreaded [1.2]

**Comment by Niilo007** [comment]
About the surgery issue (if you still need help): try the latest version of Rimthreaded, if that doesn't fix it, try replacing EPOE with EPOE fork, and tell me what happens

**Comment by wvartech** [comment]
Thanks for the reply!Yep I checked it was fixed shortly after, surgeries do work now. From: Niilo007Sent: Monday, January 25, 2021 5:35 AMTo: cseelhoff/RimThreadedCc: wvartech; AuthorSubject: Re: [cseelhoff/RimThreaded] Operations not working, and Hostile Cities with no defenders (#383) About the surgery issue (if you still need help): try the latest version of Rimthreaded, if that doesn't fix it, try replacing EPOE with EPOE fork, and tell me what happens—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by Eldulac** [comment]
**[Tested on RT 1.3.2]**

I tested this issue on several fresh games with RT and "RimCities - Citadel Update", but did not encounter any issues with settlement generation. I tested all types of settlements, from different factions.

This issue has likely been caused by load order problems and incompatibilities, of which I spotted several in the mod list.

**Conclusion**
There is no indication of problems between "RimCities - Citadel Update" and RT. The problems described in the report are likely caused by load order issues and incompatibilities, or have already been fixed.
